src/openapi/peridio-admin-openapi.yaml

```diff
@@ -275,10 +277,44 @@ x-tagGroups:
       - Releases
   - name: Events
     tags:
+      - Events
       - API Key Events
       - Device Events
       - Webhook Events
 paths:
+  /events:
+    get:
+      operationId: list-events
+      summary: List events
+      description: |
+        List all possible events (Device, Webhooks, API Key, etc.) within the specified time range. 
+        
+        By default, this time range is set to the latest 24 hours. 
+        
+        Any custom range must be within a 24-hour time window.
```

Comment:
This makes it sound like you can only ever look at the most recent 24 hours. Delete it. Will be easier to convey what you are trying to say on the param descriptions themselves.